<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioCalibrationConfig</class>
 <widget class="QWidget" name="RadioCalibrationConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>736</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>626</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>731</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;h2&gt;Radio Calibration&lt;/h2&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0,0,1,2,2,0,0">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="revLeftVCheckBox">
             <property name="text">
              <string>Rev</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>90</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGCRadioChannelDisplay" name="leftVWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGCRadioChannelDisplay" name="rightVWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>90</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="revRightVCheckBox">
             <property name="text">
              <string>Rev</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="revYawCheckBox">
             <property name="text">
              <string>Rev</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGCRadioChannelDisplay" name="yawWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>220</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>220</width>
               <height>60</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>60</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QGCRadioChannelDisplay" name="rollWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>220</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>220</width>
               <height>60</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QGCRadioChannelDisplay{
qlineargradient(spread:pad, x1:0.473, y1:0, x2:0.443, y2:0.994318, stop:0 rgba(0, 118, 0, 255), stop:1 rgba(188, 255, 192, 255))qlineargradient(spread:pad, x1:0, y1:0.494, x2:0.995074, y2:0.511, stop:0 rgba(0, 118, 0, 255), stop:1 rgba(188, 255, 192, 255))
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="revRollCheckBox">
             <property name="text">
              <string>Rev</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio5Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio6Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio7Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio8Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio9Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio10Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio11Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio12Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio13Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio14Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio15Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGCRadioChannelDisplay" name="radio16Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>150</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="elevonConfigGroupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>69</height>
          </size>
         </property>
         <property name="title">
          <string>Elevon Config</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="elevonCheckBox">
              <property name="text">
               <string>Elevons</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Elevon Output</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="elevonOutputComboBox"/>
            </item>
            <item>
             <widget class="QCheckBox" name="elevonRevCheckBox">
              <property name="text">
               <string>Elevons Rev</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="elevonCh1CheckBox">
              <property name="text">
               <string>Elevons CH1 Rev</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="elevonCh2CheckBox">
              <property name="text">
               <string>Elevons CH2 Rev</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>RC Radio</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modeComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>558</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="calibrateButton">
           <property name="text">
            <string>Calibrate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QLabel" name="label_9">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="pitchDZ_spinBox">
       <property name="maximum">
        <number>199</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="throtDZ_spinBox">
       <property name="maximum">
        <number>199</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Roll</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="writeDZ_Button">
       <property name="text">
        <string>Write to UAS</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="rollDZ_spinBox">
       <property name="maximum">
        <number>199</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="yawDZ_spinBox">
       <property name="maximum">
        <number>199</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Dead Zones</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Pitch</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Throttle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGCRadioChannelDisplay</class>
   <extends>QWidget</extends>
   <header>ui/designer/QGCRadioChannelDisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
